extensions/eclipse-che-theia-terminal/src/browser/terminal-widget/remote-terminal-widget.ts

```diff
@@ -69,14 +80,27 @@ export class RemoteTerminalWidget extends TerminalWidgetImpl {
             }
         }));
 
+        const badDefaultLoginErr = 'command terminated with exit code 126';
+        const missedPrivilegesErr = 'cannot create resource "pods/exec"';
         this.toDispose.push(this.remoteTerminalWatcher.onTerminalExecError(errEvent => {
             if (this.terminalId === errEvent.id) {
                 if (this.options.closeWidgetOnExitOrError) {
                     this.dispose();
                 }
                 this.onTermDidClose.fire(this);
                 this.onTermDidClose.dispose();
+
                 this.logger.error(`Terminal error: ${errEvent.stack}`);
+                this.channel.appendLine(errEvent.stack);
+
+                let reason = '';
+                if (errEvent.stack.indexOf(badDefaultLoginErr) !== -1) {
+                    reason = ' Possible reason is bad default login.';
```

Comment:
What exactly does "bad default login" mean?

Reply:
This means that terminal can't open a default shell while connecting to the container or maybe when user is not in `/etc/passwd`. I took this wording from the issue.
Do you have any suggestion with the message?

Reply:
According to https://stackoverflow.com/a/13046283 it's called default login shell.
my +1 to go with `default shell` or `default login shell`

Reply:
+1 for `default login shell`